qtconsole/console_widget.py

```diff
@@ -1231,10 +1242,11 @@ def callback(complete, indent):
                             try:
                                 cursor.beginEditBlock()
                                 cursor.setPosition(position)
-                                cursor.insertText('\n')
-                                self._insert_continuation_prompt(cursor)
-                                if indent:
-                                    cursor.insertText(indent)
+                                for i in range(self._is_complete_number_returns):
```

Comment:
What's this taking care of? I don't understand why it's needed as part of this PR.

Reply:
The idea was to avoid dropping returns, but it is not necessary.